packages/graph/hash_graph/lib/graph/src/logging/init.rs

```diff
@@ -46,6 +58,41 @@ where
     }
 }
 
+fn configure_opentelemetry_layer(
+    otlp_endpoint: &str,
+) -> OpenTelemetryLayer<Layered<EnvFilter, Registry>, Tracer> {
+    // Allow correlating trace IDs
+    global::set_text_map_propagator(TraceContextPropagator::new());
+    // If we need to set any tokens in the header for the tracing collector, this would be the place
+    // we do so.
+    let map = MetadataMap::new();
+
+    let pipeline = opentelemetry_otlp::new_exporter()
+        .tonic()
+        .with_endpoint(otlp_endpoint)
+        .with_timeout(std::time::Duration::from_secs(3))
```

Comment:
Do we want to make this a constant?

Reply:
I don't see a reason to make this configurable right now at least. It would be the timeout for the gRPC request I believe. Maybe bumping it to 5 seconds is fine, but I don't think we need to add an arg to configure this

Reply:
I just mean a constant at the top of the file
```
const OPENTELEMETRY_TIMEOUT_DURATION: Duration = Duration::from_secs(3);
```

Reply:
Ah gotchu, sure why not

Reply:
621932df662fcaf5a1e676b200724eb1588ba86a changed to 5 to give a little more room for any bigger trace batches